libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -346,7 +346,9 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
     private updateControlProperties(): void {
         if (this.control && this._editable) {
             this.control.id = this.id;
-            this.control.placeholder = this.placeholder;
+            if (this.placeholder && !this.control.placeholder) {
+                this.control.placeholder = this.placeholder;
+            }
```

Comment:
in cases where `fdp-form-field` does not have a placeholder but the `fdp-datetime-picker` has it, we get a blank field  or 'undefined' for the placeholder. This check only ensures that a defined/valid placeholder value is set.

Regarding the late updation part, this method is part of a change subscription as well as is called through form-field's afterViewInit, so wouldn't that ensure the value is always set?

Reply:
the current expression does not give us ability to update placeholder if this.placeholder changes second time

Reply:
Sorry but I am failing to understand why not, since the check is only checking for a valid `this.placeholder`; if it were changed also it would enter this condition, right?

Reply:
no, cause after the first time `this.control.placeholder` will have a value (asigned on the first interation) as result `this.placeholder && !this.control.placeholder` will give us `false`

Reply:
`if (this.placeholder) {
                this.control.placeholder = this.placeholder;
            }` seems to work fine with this as well. Would this change be okay?

Reply:
I think that's fine